camel/loaders/firecrawl_reader.py

```diff
@@ -68,6 +68,9 @@ def crawl(
         """
 
         try:
+            if params is None:
+                params = {}
+            params['integration'] = 'camelai'
```

Comment:
This same code appears in 4 different methods. Can we extract it to avoid repetition?

Also, this overwrites the user's integration param if they set one. Maybe use `setdefault` instead?